allennlp/models/heads/head.py

```diff
@@ -0,0 +1,21 @@
+from allennlp.models.model import Model
+
+
+class Head(Model):
```

Comment:
It would make some sense to re-write `TransformerMC` and `TransformerQA` to use this.

Reply:
Agreed, and I'm planning on doing something similar when I create a new baseline for ORB this quarter.